diff --git a/CMakeLists.txt b/CMakeLists.txt
index 02a879c..fa9a34e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,9 +49,9 @@ ELSE()
 	UNSET(VKVG_WIRED_FILL CACHE)
 	UNSET(VKVG_USE_MESA_OVERLAY CACHE)
 	IF (ANDROID)
-		ADD_COMPILE_OPTIONS(-O3 -w -ansi -pedantic)
+		ADD_COMPILE_OPTIONS(-O3 -w -pedantic)
 	ELSEIF (UNIX)		
-		ADD_COMPILE_OPTIONS(-O3 -w -ansi -pedantic -fvisibility=hidden -fno-rtti)
+		ADD_COMPILE_OPTIONS(-O3 -w -pedantic -fvisibility=hidden -fno-rtti)
 		#ADD_COMPILE_OPTIONS(-flto -pthread -lpthread -Wl,--no-as-needed)
 	ELSEIF(MSVC)
 		ADD_COMPILE_OPTIONS(/O2 /TC /W0 /Od)
